setup.py

```diff
@@ -66,7 +66,7 @@
     long_description_content_type="text/markdown",
     cmdclass=cmdclass,
     packages=setuptools.find_packages(),
-    install_requires=["jupyter_server", "nbdime ~=3.0.0b1", "packaging", "pexpect"],
+    install_requires=["jupyter_server", "nbdime~=3.0.0b1", "jupyter_packaging~=0.7.9", "packaging", "pexpect"],
```

Comment:
```suggestion
    install_requires=["jupyter_server", "nbdime~=3.0.0b1", "packaging", "pexpect"],
```

`jupyter_packaging` is not a install dependency but a setup one (only need by dev'). We should add it to the extra requires - and probably rename it as `dev` rather than `test` as there are already other packages like pre-commit or black in there.